src/main.cpp

```diff
@@ -2,7 +2,7 @@
 
 int main(int argc, char **argv)
 {
-    auto test = new std::string;
+    auto test = new std::string;  // :)
```

Comment:
warning: Value stored to 'test' during its initialization is never read [clang-analyzer-deadcode.DeadStores]
```cpp
    auto test = new std::string;  // :)
         ^
```
**/home/pajlada/git/mini-cpp-project/src/main.cpp:4:** Value stored to 'test' during its initialization is never read
```cpp
    auto test = new std::string;  // :)
         ^
```